src/os/ObjectStore.h

```diff
@@ -1029,7 +1025,7 @@ class ObjectStore {
     /// Commence a global file system sync operation.
     void start_sync() {
       Op* _op = _get_next_op();
-      _op->op = OP_STARTSYNC;
+      _op->op = 0;
       data.ops++;
     }
```

Comment:
The start_sync() method itself should be removed too.  Luckily there are no callers so no upgrade/compat issues.